### PR TITLE
New version: Bilibili.Livehime version 4.34.0.4299 [FP-D]

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Successful
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | 哔哩哔哩直播姬 | 哔哩哔哩直播姬    |
| Version     | 4.34.0.4299     | 4.34.0.4299 |
| Publisher   | 上海幻电信息科技有限公司   | 上海幻电信息科技有限公司      |
| ProductCode |           | {06BB8CDE-F4AC-4B22-A680-0F3699CB9912}_is1    |
#### Auto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) in workflow run [4408](<https://github.com/vedantmgoyal2009/vedantmgoyal2009/actions/runs/3641084599>)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

**Comment by yao-msft** [comment]
Add @zachcarp for Url Scan failures with a lot of urls, maybe we should send them in several batches?

**Comment by vedantmgoyal2009** [comment]
@denelon Please label it as blocking issue.

File: manifests/b/Bilibili/Livehime/4.34.0.4299/Bilibili.Livehime.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Publisher: 上海幻电信息科技有限公司
+PublisherUrl: https://live.bilibili.com/
+PublisherSupportUrl: https://link.bilibili.com/p/help/
+PrivacyUrl: https://www.bilibili.com/blackboard/live/mobilelink-provacypolicy.html
+Author: Shanghai Magic Electric Information Technology Co., Ltd.
+PackageName: 哔哩哔哩直播姬
+PackageUrl: https://live.bilibili.com/liveHime/
+License: Freeware
+LicenseUrl: https://www.bilibili.com/blackboard/live/activity-dLUnpNZEM1.html
+Copyright: Copyright (C) 2022 上海幻电信息科技有限公司. All Right Reserved.
```
Comment (stephengillie) [inline manifests/b/Bilibili/Livehime/4.34.0.4299/Bilibili.Livehime.locale.en-US.yaml:16]:
```suggestion
Copyright: Copyright (C) 2022 Shanghai Magic Information Technology Co., Ltd. All Right Reserved.
```

File: manifests/b/Bilibili/Livehime/4.34.0.4299/Bilibili.Livehime.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# Moniker:
+Tags:
+- broadcasting
+- live
+- live-streaming
+- livestreaming
+- screencasting
+- streaming
+# Agreements:
+ReleaseNotes: 【优化】已知问题修复
```
Comment (stephengillie) [inline manifests/b/Bilibili/Livehime/4.34.0.4299/Bilibili.Livehime.locale.en-US.yaml:29]:
```suggestion
ReleaseNotes: [Optimization] Known bug fixes
```

**Review by stephengillie — DISMISSED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**